test/functional/test_runner.py

```diff
@@ -153,6 +153,7 @@
     # These are python files that live in the functional tests directory, but are not test scripts.
     "combine_logs.py",
     "create_cache.py",
+    "example_test.py",
```

Comment:
Can we put this in the extended scripts, just to make sure it has correct syntax?